splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -129,16 +129,26 @@ private void configureHandler() throws Exception {
             // Transform message to add splunk wrapper to the json
             .transform().simple("{\"source\": \"eventing\", \"sourcetype\": \"Insights event\", \"event\": ${body}}")
 
+            // Redirect depending on http or https (different default ports) so that it goes to the default splunk port
             // Send the message to Splunk's HEC as a splunk formattted event.
             // It sends token via Basic Preemptive Authentication.
             // POST method is being used, set up explicitly
             // (see https://camel.apache.org/components/latest/http-component.html#_which_http_method_will_be_used).
-            .toD(http("$simple{headers.metadata[url]}/services/collector/event")
-                .authenticationPreemptive(true)
-                .authMethod("Basic")
-                .httpMethod("POST")
-                .authUsername("x")
-                .authPassword("$simple{headers.extras[token]}"))
+            .choice()
+                .when(simple("${headers.metadata[url]} startsWith 'http://'"))
+                    .toD(http("$simple{headers.metadata[url]}/services/collector/event")
+                        .authenticationPreemptive(true)
+                        .authMethod("Basic")
+                        .httpMethod("POST")
+                        .authUsername("x")
+                        .authPassword("$simple{headers.extras[token]}"))
+                .otherwise()
+                    .toD(https("$simple{headers.metadata[url]}/services/collector/event")
```

Comment:
It does not work with https urls like `https://abcd-000-1111-22-33.ngrok.io/`:
```
KafkaConsumer[platform.notifications.tocamel]) Fail with for id 6aba6454-20bf-4677-8d93-e41b326a2212 : https: Name or service not known
```

Reply:
@CodeHeeler, so the issue is with the `https`/`http` components, as they are expecting URI without the protocol. We can take the protocol out of the `metadata[url]`. 

Good thing is that we can call [Java String methods](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html) within the simple. For example here we can use [`replaceFirst`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#replaceFirst(java.lang.String,java.lang.String)).